### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by msachi** [marisid/handlebars-intro-workshop#1]
Looks great, just a few minor suggestions from my side:

1) Missing link here?
 > Image Cred: Real Python from this medium post
2) 'both extensions can be equally used' sounds a bit strange - maybe 'either extension can be used?' Might also be nice to add that using hbs has the advantage of syntax higlighting in Atom.
3) I realized that `process.env.HOST` is not actually set by deafult on Heroku, unless you manually specify the `HOST` variable on Heroku. Maybe omit it to avoid confusion (I will omit it from my walk-through).
4) What is meant by this?
`{{! Index view can be seen as a wildcard for any other view.}}`

**Comment by marisid** [comment]
Thorough review as always @msachi - thanks thanks thanks! 

I've fixed everything you suggested, replaced `.html` with `.hbs` file extension and updated master branch. 

Regarding point 4, all I wanted to say was that `index.hbs` and really any file sitting immediately under views is like a wildcard that can be viewed/rendered in conjunction with the layout. But I guess this is confusing - so I just removed it and I will explain it in the code along :smile:
